gluten-ut/spark33/src/test/scala/org/apache/gluten/utils/clickhouse/ClickHouseTestSettings.scala

```diff
@@ -692,11 +692,6 @@ class ClickHouseTestSettings extends BackendTestSettings {
       "SPARK-36740: ArrayMin/ArrayMax/SortArray should handle NaN greater then non-NaN value")
     .excludeGlutenTest("Shuffle")
   enableSuite[GlutenComplexTypeSuite]
-    .exclude("SPARK-33386: GetArrayItem ArrayIndexOutOfBoundsException")
```

Comment:
how about spark 32 and spark 35 ?

Reply:
It could also be removed in spark 32 and 35

Reply:
done